SDL_VERSION_ATLEAST is not defined

/build/mgba-master/src/platform/qt/main.cpp:33:27: error: function-like macro 'SDL_VERSION_ATLEAST' is not defined
#if defined(BUILD_SDL) && SDL_VERSION_ATLEAST(2, 0, 0)

I am on OSx Sierra 10.12.5.

I honestly don't know what information would be useful for debugging this. I ran the brew install successfully. SDL2 is definitely installed. I searched around the repo and it doesn't look like SDL_VERSION_ATLEAST is defined anywhere, but I am not a C guy.

Any guidance is much appreciated!!!!!!

--- a/README.md
+++ b/README.md
@@ -122,7 +122,7 @@ This will build and install mGBA into `/usr/bin` and `/usr/lib`. Dependencies th
 
 If you are on macOS, the steps are a little different. Assuming you are using the homebrew package manager, the recommended commands to obtain the dependencies and build are:
 
-	brew install cmake ffmpeg imagemagick libzip qt5 sdl2 libedit
+	brew install cmake ffmpeg imagemagick libzip qt5 sdl2 libedit pkg-config
 	mkdir build
 	cd build
 	cmake -DCMAKE_PREFIX_PATH=`brew --prefix qt5` ..

--- a/README_DE.md
+++ b/README_DE.md
@@ -122,7 +122,7 @@ Damit wird mGBA gebaut und in `/usr/bin` und `/usr/lib` installiert. Installiert
 
 Wenn Du macOS verwendest, sind die einzelnen Schritte etwas anders. Angenommen, dass Du eine Homebrew-Paketverwaltung verwendest, werden folgende Schritte zum installieren der Abhängigkeiten und anschließenden bauen von mGBA empfohlen:
 
-	brew install cmake ffmpeg imagemagick libzip qt5 sdl2 libedit
+	brew install cmake ffmpeg imagemagick libzip qt5 sdl2 libedit pkg-config
 	mkdir build
 	cd build
 	cmake -DCMAKE_PREFIX_PATH='brew --prefix qt5' ..

